concent_api/core/views.py

```diff
@@ -271,6 +288,72 @@ def handle_send_force_get_task_result(client_message: message.concents.ForceGetT
         )
 
 
+def handle_send_force_subtask_results(client_message: message.concents.ForceSubtaskResults):
+    assert client_message.TYPE in message.registered_message_types
+
+    current_time = int(datetime.datetime.now().timestamp())
+
+    if Message.objects.filter(
+        type    = client_message.TYPE,
+        task_id = client_message.ack_report_computed_task.task_to_compute.compute_task_def['task_id']
+    ).exists():
+        return message.concents.ServiceRefused(
+            timestamp   = client_message.timestamp,
+            reason      = message.concents.ServiceRefused.REASON.TaskAlreadyProcessed,
+        )
+
+    provider_has_enough_funds = get_provider_account_status(client_message)
```

Comment:
I know this is temporary, but two things here:
1. If method is returning True/False its name should start with `is_` or `has_`.
2. `provider_has_enough_funds` is not used anywhere later so you can call it directly in `if`.

Reply:
:+1: